### PR TITLE
GitHub-to-AWS keyless authentication

### PR DESCRIPTION
**Pull request by alliington**
**Checklist**
I’ve used the pull request template
I've linked this PR to the relevant issue (if mission work)
I’ve written unit tests for these changes (if code change)
I’ve update the documentation in
**README.md**
Elsewhere (plese link)

**git push --set-upstream origin github_actions4aws_role**
Enumerating objects: 6, done.
Counting objects: 100% (6/6), done.
Delta compression using up to 8 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (5/5), 1000 bytes | 1000.00 KiB/s, done.
Total 5 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
remote: 
remote: Create a pull request for 'github_actions4aws_role' on GitHub by visiting:
remote:      https://github.com/alphagov/security.txt/pull/new/github_actions4aws_role
remote: 
To github.com:alphagov/security.txt.git
 * [new branch]      github_actions4aws_role -> github_actions4aws_role
branch 'github_actions4aws_role' set up to track 'origin/github_actions4aws_role'.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+     role-to-assume: arn:aws:iam::576485064911:role/github_actions_role
+     role-session-name: Github
+     aws-region: ${{ env.AWS_REGION }}
+    
+       - name: Deploy and update makefile
+        path: main
```
Comment (bartlabno) [inline .github/workflows/deploy.yml]: looks like yaml file formatting issue here and in the next lines as well?

**Review by elioffice — DISMISSED** [review]

**Review by bartlabno — CHANGES_REQUESTED** [review]
Please check yaml formatting

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+on:
+  workflow_dispatch:
+  push:
+    branches:    
+      - 'main'
+    paths:
+      - 'security.txt'
+      - 'thanks.txt'
+  schedule:
+     - cron: "0 2 * * MON-FRI" # Runs at 02:00 UTC
```
Comment (bartlabno) [inline .github/workflows/deploy.yml]: you are missing one more star at the end of the cron - https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (alliington) [reply .github/workflows/deploy.yml]: Thanks Bart, 
Re: looks like yaml file formatting issue here and in the next lines as well
could you kindly provide a solution.
Regards,
Adebola.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (alliington) [reply .github/workflows/deploy.yml]: Many thanks and I will review this.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    run:
+        shell: bash
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+     role-to-assume: arn:aws:iam::576485064911:role/github_actions_role
```
Comment (bartlabno) [inline .github/workflows/deploy.yml]: yaml formatting issue here

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (bartlabno) [reply .github/workflows/deploy.yml]: yaml has very specific requirements about linting. ie child must be tabbed properly

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (alliington) [reply .github/workflows/deploy.yml]: Thanks Bart,  let's articulate and Incorporate the above at our meeting slated for  15:30pm this afternoon.
**Review of Pull request for - GitHub-to-AWS keyless authentication**
Tuesday, 20 February · 3:30 – 4:30pm
Time zone: Europe/London
Google Meet joining info
Video call link: https://meet.google.com/eav-pcew-crg
Or dial: ‪(GB) +44 20 3956 2045‬ PIN: ‪175 468 683‬#
More phone numbers: https://tel.meet/eav-pcew-crg?pin=8561866440643
**Regards,
Adebola.**

**Review by Wa1idMahmoud — APPROVED** [review]

**Review by bartlabno — APPROVED** [review]

**Pull request merged**